### ISSUE TITLE
sculpt: monitor routing changes in config/deploy

### ISSUE THREAD

**Issue by nfeske** [genodelabs/genode#2912]
As discussed on the mailing list (https://lists.genode.org/pipermail/users/2018-July/006220.html), the current version of the sculpt manager evaluates the `<route>` information of `<start>` nodes before starting a component but it does not re-evaluate the condition during the lifetime of the component. It would be nice to defer the restart of a component with a route changed to an unavailable server, similar as to how the initial start of the component is deferred.
